### ISSUE TITLE
Changelog file

### ISSUE THREAD

**Issue by vogloblinsky** [dojo/framework#227]
Please create and use a Changelog file for following updates.

Thanks

**Comment by vogloblinsky** [comment]
Yes i have seen that but the standard in OSS is to have this kind of changelog file, even for minor and patch versions.

**Comment by matt-gadd** [comment]
@vogloblinsky sorry for the delayed reply. we will issue a CHANGELOG file sometime in the future i think 👍

**Comment by vogloblinsky** [comment]
ok

**Comment by matt-gadd** [comment]
@vogloblinsky going to leave open to track if thats okay?
